### ISSUE TITLE
Population selections

### ISSUE THREAD

**Issue by cnxiaobo94** [stschiff/msmc2#49]
Hi, I'm doing some population demographical analysis among three populations, and I have two questions I'd like to ask for help.
1. There is only one individual in population A, and whether it can be analyzed?
2. If the previous question is yes, can the three populations (A, B, C) be simulated at the same time (in one command line), or do simulations in pairs (A-B, A-C, B-C)?
Looking forward to your reply! Thanks!

**Comment by tcztzy** [comment]
1. As far as I know, you can't analyze "population" with only one individual, it doesn't make sense, "I suggest you don't actually run this because we won't need this single-sample processing". Actually, `MSMC2` recommends you provide trio (three individuals, parents and their offspring).
2. `msmc2` could take more than one `.multihetsep.txt` file. You can write your own shell script to run cross-population estimate. 
3. [`msmc-tools`](https://github.com/stschiff/msmc-tools/) provided a [friendly guide](https://github.com/stschiff/msmc-tools/blob/master/msmc-tutorial/guide.md) for very beginner, if you still have `HowTo` problems, maybe that repo is a good place to open issue.

**Comment by stschiff** [comment]
> Hi, I'm doing some population demographical analysis among three populations, and I have two questions I'd like to ask for help.
> 
> 1. There is only one individual in population A, and whether it can be analyzed?

This would then be like in PSMC. That is possible, and would give you effective population size estimates for the population from which this sample is from.

> 2. If the previous question is yes, can the three populations (A, B, C) be simulated at the same time (in one command line), or do simulations in pairs (A-B, A-C, B-C)?

I don't know what you mean by "simulate". If you mean "analyse", then no, MSMC/MSMC2 cannot analyse three populations at the same time. But you could estimate separately:

- coalescence rates within populations A, B and C
- cross-coalescence rates between populations A-B, A-C and B-C.

**Comment by tcztzy** [comment]
> > Hi, I'm doing some population demographical analysis among three populations, and I have two questions I'd like to ask for help.
> > 
> > 1. There is only one individual in population A, and whether it can be analyzed?
> 
> This would then be like in PSMC. That is possible, and would give you effective population size estimates for the population from which this sample is from.
> 
> > 2. If the previous question is yes, can the three populations (A, B, C) be simulated at the same time (in one command line), or do simulations in pairs (A-B, A-C, B-C)?
> 
> I don't know what you mean by "simulate". If you mean "analyse", then no, MSMC/MSMC2 cannot analyse three populations at the same time. But you could estimate separately:
> 
> * coalescence rates within populations A, B and C
> * cross-coalescence rates between populations A-B, A-C and B-C.

If the individual is homozygote (like rice and other self-pollinated plant), can it still be analyzed? I tried but it seems Powell minimization exceeding maximum iterations.

**Comment by stschiff** [comment]
No, you can't analyze a homozygous genome alone. You can of course treat a homozygous genome as a single haplotype and combine it with other such genomes to create something like a pseudo-diploid to estimate coalescence rates between them.

**Comment by tcztzy** [comment]
> No, you can't analyze a homozygous genome alone. You can of course treat a homozygous genome as a single haplotype and combine it with other such genomes to create something like a pseudo-diploid to estimate coalescence rates between them.

Thank you for your patient, it helps me.
